atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceImpl.java

```diff
@@ -402,7 +402,7 @@ private static ExecutorService createInstrumentedFixedThreadPool(CassandraKeyVal
             MetricRegistry registry) {
         return new InstrumentedExecutorService(
                 createFixedThreadPool("Atlas Cassandra KVS",
-                        config.poolSize() * config.servers().size()),
+                        config.poolSize() * config.servers().thrift().size()),
```

Comment:
this usage feels somewhat awkward, perhaps put `size()` on `CassandraServersConfig`, although not sure how I feel about this

Reply:
`size` or `numberOfHosts`